operator/images/cluster-setup/content/bin/install.sh

```diff
@@ -74,17 +75,29 @@ parse_args() {
     esac
     shift
   done
+  DEBUG="${DEBUG:-}"
+}
+
+init() {
+  local tmp_workspace_dir="/tmp/workspace"
+  if [[ -e "$tmp_workspace_dir" ]]; then
+    rm -rf "$tmp_workspace_dir"
+  fi
+  cp -rf "$WORKSPACE_DIR" "$tmp_workspace_dir"
+  WORKSPACE_DIR="$tmp_workspace_dir"
 }
 
 # turns off tracing even with set -x mode enabled across the script to prevent secrets leaking
 setx_off() {
   set +x
-} 2>/dev/null
```

Comment:
the reason I put these is that users do not need to know that we are turning on and turning off tracing. Removing this will print set +x and set -x which is unnecessary I feel.